app/code/Magento/InventoryBundleProduct/Test/_files/product_bundle_rollback.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\CatalogInventory\Model\Stock\Status;
```

Comment:
why do we use legacy CatalogInventory APIs to manage inventory of Bundle product?